it/it-tests/src/test/java/util/ItUtils.java

```diff
@@ -338,9 +341,35 @@ public static ComponentNavigation getComponentNavigation(Orchestrator orchestrat
     return ComponentNavigation.parse(content);
   }
 
-  public static File findFileInClasspath(String classpathRelativeFilename) throws URISyntaxException {
-    ResourceLocation location = FileLocation.ofClasspath("/organization/IssueAssignTest/one-issue-per-file-profile.xml");
-    return new File(ItUtils.class.getResource(location.getPath()).toURI());
+  public static void restoreProfile(Orchestrator orchestrator, String classpathRelativeFilename) {
```

Comment:
The String parameter is misleading. I suggest to replace it by URL so that caller does simply:
```
restoreProfile(orchestrator, getClass().getResource("FooTest/profile.xml"));
```
(or `getClass().getResourceAsStream("FooTest/profile.xml")`). That would allow to remove the method `findFileInClasspath()` and to allow setting byte[] or Reader types to `RestoreWsRequest`.